btrace-agent/src/main/java/org/openjdk/btrace/agent/Main.java

```diff
@@ -453,14 +454,22 @@ private static void parseArgs() {
             }
             break;
           }
-        case SCRIPT_OUTPUT_DIR:
+          case SCRIPT_OUTPUT_DIR:
           {
             if (!p.isEmpty()) {
-              settings.setOutputDir(p);
+              settings.setScriptOutputDir(p);
               log.debug(SCRIPT_OUTPUT_DIR + " is {}", p);
             }
             break;
           }
+        case SCRIPT_DIR:
+          {
+            if (!p.isEmpty()) {
+              settings.setOutputDir(p);
```

Comment:
This does not look right - it should rather be `setScriptDir(p)`. In fact, `get/setOuptutDir()` should be removed as it is replaced by `get/setScriptOutputDir()` (you would need to check where `getOutputDir()` is used and replace it with `getScriptOutputDir()`)